src/structure_managers/structure_manager_collection.hh

```diff
@@ -243,6 +243,12 @@ namespace rascal {
       return this->managers[index]->get_shared_ptr();
     }
 
+    /**
+     * get the dense feature matrix associated with the property built
+     * with calculator in the elements of the manager collection.
```

Comment:
I don't understand this part. could you expand a bit?

Reply:
it means that the size of the output array (nb of columns) depends on the species present in the structures held by manager_collection.